compiler/src/codegen/compcore.re

```diff
@@ -273,27 +293,12 @@ let init_codegen_env = name => {
   required_imports: [],
 };
 
-/** Static runtime values */
-
-// Static pointer to the runtime heap
-// Leaves low 1000 memory unused for Binaryen optimizations
-let runtime_heap_ptr = ref(Grain_utils.Config.default_memory_base);
-// Static pointer to runtime type information
-let runtime_type_metadata_ptr = () => runtime_heap_ptr^ + 0x08;
-
 let reset = () => {
   reset_labels();
   List.iter(
     imp => imp.mimp_used = imp.mimp_mod == grain_env_mod,
     runtime_imports,
   );
-  runtime_heap_ptr :=
-    (
-      switch (Grain_utils.Config.memory_base^) {
```

Comment:
The memory_base config still exists? If so, what does it change? Should it be renamed/reworded?

Reply:
Yes, and it does the same exact thing as before—allowing you to reserve some upfront memory in your wasm module. It just no longer has an effect on the static runtime pointers (that no longer exist).